spec/fixtures/vcr_cassettes/dfc-context.yml

```diff
@@ -2,7 +2,7 @@
 http_interactions:
 - request:
     method: get
-    uri: http://static.datafoodconsortium.org/ontologies/context.json
+    uri: https://www.datafoodconsortium.org/
```

Comment:
The DFC changed the URL of the context. It was on static.datafoodconsortium.org and it's on Github. But in order to have a stable URL they decided to use the homepage and put the context location in a header field when you request the homepage. That is common practice in the Semantic Web world. The home page is more like a name for the DFC saying: "use the context of the DFC (and the URL will tell you somehow where to find it)".

I'm actually planning on scanning documents for that URL and replacing it with our stored context so that we don't have to do a network request at all. The context shouldn't change much and it's fair enough to have a cached version. It just means that you "know" the DFC and you can work with its data without looking up further URLs.

It will be possible to submit a document referring to a different, custom context. As long as it's compatible with the DFC context, it will work. But in that case we have to look it up unless it's embedded in the document.

Reply:
👍  Thanks for explaining!